documentation/getting-started.md

```diff
@@ -1,64 +1,29 @@
 # Get started
 
-## Step 1: Install the barista-components and Angular CDK
+## TLDR;
```

Comment:
I would remove this section. The page is not that long that you need a `TLDR;` as the command can be found again in the next line.